### DISCUSSION TITLE
Silent mode (no console output)

### DISCUSSION THREAD

**Discussion by zm-cttae** [eslint/eslint#17693] [Ideas] [answered]
I want to use `eslint --fix` without any errors being piped to the console.
In essence like an `option.logLevel` of `"off"` - and if you feel this is better, that's welcome.

I'm proposing that this is done using a new top-level option of `--silent`.

**Comment by Rec0iL99** [comment]
Hi @zm-cttae, your proposal is not clear enough. I believe `eslint --fix` will try to fix as many issues as possible and only the unfixed issues are outputted to console.

**Comment by chiefmikey** [comment, marked as answer]
@zm-cttae if interested in a workaround depending on the system an option is to feed the execution output to null. For example if running `eslint --fix` from a package.json script you could use either `npm run fix > /dev/null 2>&1` and run with absolutely no output, or add it directly to the script like `eslint --fix > /dev/null 2>&1` but it will still output any npm messages (if using npm). Similarly if you'd like to allow only error messages to print you could redirect the standard output only like `eslint --fix > /dev/null`

**Reply by zm-cttae** [reply]
I did find an alternative workaround using a custom formatter that emits an empty string:

https://github.com/vsce-toolroom/vscode-textmate-languageservice/blob/v3.0.0/scripts/silent-formatter.js

https://github.com/vsce-toolroom/vscode-textmate-languageservice/blob/v3.0.0/scripts/pegjs.js#L49-L51
